include/picongpu/particles/ParticlesInit.kernel

```diff
@@ -105,7 +105,7 @@ namespace picongpu
         DINLINE void
         operator()(
             T_DensityProfile densityFunctor,
-            T_PositionFunctor const & positionFunctor,
+            T_PositionFunctor positionFunctor,
             DataSpace< simDim > const & totalGpuCellOffset,
```

Comment:
note: preparing for this being the entry function of a kernel, the other params should not be references either